fhir-config/src/main/java/com/ibm/fhir/config/PropertyGroup.java

```diff
@@ -292,6 +292,8 @@ public static Object convertJsonValue(JsonValue jsonValue) throws Exception {
         case FALSE:
             result = Boolean.FALSE;
             break;
+        case NULL:
```

Comment:
This method can return null now. There are loads of places in the code where we don't check for null when calling PropertyGroup#getValue(), so we'd just end up with a NPE if things were misconfigured.

Reply:
I ended up doing the above only for getProperties (and not for convertJsonArray).  IMHO that should be sufficient, but let me know if you think we should do something for convertJsonArray as well...

Reply:
sounds good